core/src/com/mygdx/game/event/events/PlayerMovedEvent.java

```diff
@@ -0,0 +1,37 @@
+package com.mygdx.game.event.events;
+
+import com.badlogic.gdx.math.Vector2;
+import com.mygdx.game.constant.State;
+import com.mygdx.game.event.Event;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Value;
+
+/**
+ * Event containing info about a remote player. Also used to send the local player's info to the server.
+ */
+@Value
+@Builder
+public class PlayerMovedEvent implements Event<PlayerMovedEvent.PlayerMovedPayload> {
```

Comment:
Rename this, this will get raised if the "player" gets updated

Reply:
Will be taking this up as a separate change since it requires changes in the server as well. 
Created an issue to track this : https://github.com/sagdutt/Raze-LibGdx/issues/5